app/javascript/components/payment-error-button.vue

```diff
@@ -32,24 +39,80 @@
 <script>
 
 import invoiceApi from '../api/invoices';
+import { mapState, mapGetters, mapActions } from 'vuex';
+import utils from '../utils/';
```

Comment:
'utils' is defined but never used                                      no-unused-vars